DragonFruit.Link/Game/Extensions/SteamGlobalGameStatsExtensions.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading;
```

Comment:
looks like this is missing? if it's incompatible with `params` change it to a plain `IEnumerable<string>`